code/modules/detectivework/tools/bullet_scanner.dm

```diff
@@ -2,7 +2,8 @@
 
 /obj/item/device/bullet_scanner
 	name = "\improper Bullet Scanner"
-	desc = "A small device used to scan bullet holes or laser marks to get more data on the gun that fire it."
+	desc = "A small device used to scan bullet holes or laser marks to get more data on the gun that fired it."
+
```

Comment:
what's up with this newline

Reply:
Someone made typo. i changed it from gun that fire it to gun that fired it

Reply:
actually i thought it already was PRed with other branch, but oh well